### DISCUSSION TITLE
@moonrepo need more marketing

### DISCUSSION THREAD

**Discussion by ImBIOS** [moonrepo/moon#1262] [General]
My reaction after reading this: https://moonrepo.dev/docs/comparison was, WTF. There's a monorepo management tool out there better than turbopack, nx, lerna, etc. in each field that my projects reaaallllly need.

I think @moonrepo team needs to pump up its marketing budget, so, this great software is getting more and more support from the community.

**Comment by milesj** [comment]
Thanks for the kind words! I definitely agree, but this project is just 2 of us, mostly me, so there's no budget for marketing, and I don't have enough time to push it. I'm also holding off a bit until 2.0, since I plan to fix a lot of the headaches that users have been having.

With that said, I would definitely appreciate it of the community can help drive this forward a bit!
